src/System.Text.Json/src/System/Text/Json/JsonParser.cs

```diff
@@ -0,0 +1,565 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Text.Json
+{
+    public struct JsonParser : IDisposable
```

Comment:
JsonParser actually reads/parses through the json string and builds the DB in the buffer. Is this the type of information you were looking for? I will add the issues, sure.